### PR TITLE
Phytium: Some basic information of Phytium FTC series cpu

### PR DESCRIPTION
**Pull request by JiakunShuai**
Add some basic information and description of Phytium FTC series cpus.

### CODE REVIEW

File: arch/arm64/include/asm/cputype.h [hunk 1]
```diff
@@ -61,6 +61,7 @@
 #define ARM_CPU_IMP_HISI		0x48
 #define ARM_CPU_IMP_APPLE		0x61
 #define ARM_CPU_IMP_AMPERE		0xC0
+#define ARM_CPU_IMP_PHYTIUM        0x70
```
Comment (opsiff) [inline arch/arm64/include/asm/cputype.h]: 修改一下缩进

Comment (JiakunShuai) [reply arch/arm64/include/asm/cputype.h]: 已修改了

**Review by opsiff — APPROVED** [review]

**Comment by Avenger-285714** [comment]
it's ok. thanks.

**Pull request merged**